injection/sharedmain/main.go

```diff
@@ -120,12 +119,14 @@ var (
 // by name.
 func MainNamed(ctx context.Context, component string, ctors ...injection.NamedControllerConstructor) {
 
-	disabledControllers := pflag.StringSlice("disable-controllers", []string{}, "Comma-separated list of disabled controllers.")
-
+	var (
+		disabledControllers string
+	)
 	// HACK: This parses flags, so the above should be set once this runs.
```

Comment:
This comment should be before like 127 `cfg := injection.ParseAndGetRESTConfigOrDie()`